src/components/Task.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import '../css/index.css'
+import { Tasks, useReminders } from '../utils/RemindersContext'
+import calendar from '../assets/icons/calendar-two.png'
+import check from '../assets/icons/checkmark.png'
+import flag from '../assets/icons/image.png'
+
+const Task = ({ id, input, flagged }: Tasks) => {
+  const {
+    activeList,
+    setLists,
+    lists,
+    activeListId,
+  } = useReminders();
+
+  if (!activeList) {
+    return <main className="w-[65%] ml-16 mt-8" />;
+  }
+
+  return (
+    <div className="md:ml-6 mt-6 md:w-[46%] flex flex-row">
+      <button
+      type="button"
+      onClick={() => setLists(lists.map(list =>
+        list.id === activeListId
+          ? {
+            ...activeList,
+            tasks: activeList.tasks.filter(
+              task => task.id !== id
+            ),
+          }
+          : list)
+      )}
+        className="cursor-pointer hover:line-through text-[18px] md:text-[22px]"
+      >
+      <li>
+        {input}
+      </li>
+      </button>
+      <button type="button" className="ml-auto mr-6 w-36 flex p-2 border-2 border-slate-100 rounded-lg items-cente bg-slate-100">
+        <img alt="icon" className="h-4 pl-2" src={calendar} />
+        <input
```

Comment:
`<input` inside of a `<button` is a bad idea, especially because `<button` is not actually doing anything? It could just be a `div`